app/ts/library/utilities.ts

```diff
@@ -20,3 +22,30 @@ export function removeNonStringsAndTrim(...strings: (string | boolean | undefine
 			.join(' ')
 	)
 }
+
+/**
+ * Check string for valid address format
+ */
+export function assertsAddress(string: string): asserts string is `0x${string}` {
```

Comment:
we can call `ethers.utils.isAddress` here

Reply:
I replaced this with `parseAddress`, which uses `ethers.utils.getAddress` as validator, as the `0x${string}` type needs to be satisfied for addresses

Reply:
I removed `parseAddress` on the latest commit after changing the method used to get the active account to `signer.getAddress()`